Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN mkdir -p /app
 WORKDIR /app
 
 ENV DEBIAN_FRONTEND=noninteractive
+# https://create-react-app.dev/docs/advanced-configuration/
+ENV INLINE_RUNTIME_CHUNK=false
```

Comment:
Is this the correct way of setting `env` variable in Docker? This flag should work like setting it in `.env` file (worked on build locally).

Reply:
This will set the environment variable during the build only. So `yarn install` will see it when the image is built but it won't be set when the resulting image is run.